v2/generator/execute.go

```diff
@@ -210,9 +210,16 @@ func (c *Context) ExecutePackage(p Package) error {
 		return fmt.Errorf("no source-path for package %s", p.Path())
 	}
 	klog.V(5).Infof("Processing package %q, disk location %q", p.Name(), path)
+
 	// Filter out any types the *package* doesn't care about.
 	packageContext := c.filteredBy(p.Filter)
-	os.MkdirAll(path, 0755)
+
+	if !c.Verify {
+		if err := os.MkdirAll(path, 0755); err != nil {
```

Comment:
This is stylistic preference, but I like `0o755` to make octal values really explicit (same as hex and binary).

Reply:
I didn't even know that was a thing.  IMO "0755" is UNIX and C since...forever.  Everyone knows what that means.